Jenkinsfile

```diff
@@ -101,6 +102,23 @@ node('ubuntu-zion') {
         gitHub.statusUpdate commitId, 'success', 'test', 'Tests succeeded'
       }
     }
+    stage('Evaluate') {
+      
+      //Create tar of our image
+      OsTools.runSafe(this, "docker save ${imageName} -o ${env.WORKSPACE}/${tarName}")
+      
+      //decide which stage we are creating
+      def theStage = 'build'
+      if(branch == 'master') {
+        theStage = 'release'
+      }
```

Comment:
It's just a style thing so feel free to disregard, but I feel like we usually use the ternary operator for this sort of thing.

```groovy
def theStage = branch == 'master' ? 'release' : 'build'
```

Reply:
Fixed in b6aaba7